.travis.yml

```diff
@@ -11,5 +11,6 @@ script:
 deploy:
   provider: heroku
   api_key:
-    secure: b+wMjWhVQDDKj5YLJbMclzCHPv+j22gqo0OL5HOvrIMEb7GKlr5Mj5CPgDf+Awy2GzsCF1l2taA4RGtqmnFctAKmpUorXKiF+gvNPyYrZNAJxUtVSq1570n/HSboDGtrSzRVr95BMbvoEXqC3RprAESnz9uycUxi0PIs06Ou70PCSuRvkVVmxl7VMYCWGCFaXkwrUhkeuHoaQJaY0jSYG/KLV6O1JyHnqrO81TtuI/BDq7zvz1mrP2mlAM/liSvxL7eN2r475sf5/FcfGu3OUvxfh+byZo0ItyniPZyyHMXNTwpN8TtqjEDaJdWiyOL28wvTJvgHvlW+ikZIs4GFrvaocnkxyYjakXhkLh4GNO8cFFML4C1nsrnOmbuTUJzQ1WanAiTlqMYltE5GyGv4e56qKhddbeoFG6MeKAgUk3bW/pqdfBKbHnx09Ggme18qj5bS7fLu8orHGCYag2s8rjWtB+dRBlYYogQh/4r6yM9TOTMP+ch7ntF0C04vlumjSDGy/jpwbgfPja0ufPl3ZQZdR5xvbzI9Bw6SXIH9Nxo2Jj2Aa+a2cm951iQPnxRe319QBjtio70r8N0uTBwM4zlSf9mBOkaT1cIk72SlppOgc9YkKZ9spZcp6YKm2GfZHPBx4oCJEO91Qfa/mmCvxUdwTYY8588N4cYYA2g/6i0=
-    app: zeroandbeyond
+    secure: lUIuDVxGzkgAYOzdzbJlHdFr8utUn5lA2A4/a1XNCL5gpwNsvLQfKg3Q6DjEj9Lzez6AMGSn9MRrd8JAAanatkYtEwer6kzv3yJgGgcVbU6RIm99YsJ7rPGzSoFPt7f6KdyIUlrCPtqxZscsTzTQOGhNL3CPB/JaB9SSoeXj9pEhacM4OJv0mioL7/dYrPNGURbFsbl4P5byH8WtYYGybh+7Rr7yLlMqE6vFBCos4mwICdw3WLHkGy8Q/oH1wZtXRLr4ZqB851o7roSd9/DPg4k7E/FG0b2CLzD2iPETkOyxf7T2y9KZcTkmrJLwT5WuAWm4rb88v/e/1aU/lWUEaz+UnxCSjDcqPW4L4KI/I/YdQVkDEAAC2e1eqqVKR5VBVxMw26xOfhznsLwg1EqW9qpv3FnhAp9poV30u7rcrv0T+31TOLxCpNSQbPF39ZXOtNv7h2BFGWKqcdWzcGj6+RJQBySQ/0DlIEaJnY41SWCFRWIG5V8PP8LV29ugcJNYnkHJ5STtFtldo6Ie5yFn8jyNRVBaVkiMt5CyrNtGnfLdAU/t3mNMiA6B9GVPZrukFWbE7XaQkd37BXZq48tcavMrlBdDt/JvGHCA9x1y9KBCfQnRq6dYa1HGEGp5NkjNSIu62vI27rqHXhyXnKiLWrFh4LKYD3lVJseDAJBneO0=
```

Comment:
@hillstew Do you need to add `app:  zeroandbeyond` and  possibly `deploy: skip_cleanup: true`?

See this Travis CI discussion: https://travis-ci.community/t/no-stash-entries-found-and-api-request-failed/119

Reply:
I had it and it failed 😞 but maybe the `skip_cleanup`. I will make it as a suggestion and then would you approve it?

Reply:
Approved!

Reply:
Approved, I will merge the PR.